lib/controller.js

```diff
@@ -68,7 +72,7 @@ Tessel.list = function(opts) {
             // Report that selected Tessel to the user
             logs.info('Multiple Tessels found.');
             if (tessel) {
-              logs.info('Will default to', tessel.name, '.');
+              logs.info('Will default to', tessel.name + '.');
```

Comment:
This should work and looks nice:

```javascript
logs.info('Will default to %s.', tessel.name);
```

It uses `util.format` which is a builtin in Node that does this. That works with `console.log` as well :+1: